dora/stats/admin.py

```diff
@@ -151,12 +152,36 @@ class OrientationEventAdmin(AnalyticsEventAdmin):
     list_display = [
         "date",
         "service",
+        "is_di",
         "structure",
         "structure_department",
         "user",
         "anonymous_user_hash",
     ]
     list_filter = [
+        "is_di",
+        "date",
+        "structure_department",
+    ]
+    filter_horizontal = [
+        "categories",
+        "subcategories",
+    ]
+
+
+class ServiceShareAdmin(AnalyticsEventAdmin):
+    raw_id_fields = ("service", "structure", "user")
```

Comment:
La voilà la petite lenteur :) 
```suggestion
    raw_id_fields = ("service", "structure", "user", "search_view")
```

Reply:
Bien vu !